charts/plugin-health-scoring/values.yaml

```
@@ -45,14 +45,15 @@ resources: {}
 #   cpu: 100m
 #   memory: 128Mi
 
-probes:
-  path: /actuator/health
 nodeSelector: {}
 tolerations: []
 affinity: {}
+probes:
+  readiness: /actuator/health/readiness
+  liveness: /actuator/health/liveness
```

```suggestion
probes:
  path: /actuator/health
  readinessPath: /actuator/health/readiness
  livenessPath: /actuator/health/liveness
```
You need to keep the default path in the values.